bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -491,13 +487,11 @@
 		<state readOnly="true"/>
 	</channel-type>
 
-
 	<thing-type id="airqualitysensor">
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="deconz"/>
 		</supported-bridge-type-refs>
-		<label>Air quality Sensor</label>
-		<description>An air quality sensor</description>
+		<label>Carbon-monoxide Sensor</label>
```

Comment:
@J-N-K - late finding: Was this change intended? This is the same label as for thing type `carbonmonoxidesensor`.

Reply:
Yes, when the description does not provide additional information it can be omitted. I also remove the article because it is not needed.

Reply:
Fixed in #14636